community/samlalwani/Ibis_H3_Example/Ibis_H3_Example.py

```diff
@@ -11,19 +11,11 @@ def udf(bbox=None, resolution: int = 9, min_count: int = 10):
     from ibis import _
 
     # DuckDB is only used to download extension
-    utils = fused.load(
-        "https://github.com/fusedio/udfs/tree/f928ee1/public/common/"
-    ).utils
-    h3_utils = fused.load(
-        "https://github.com/fusedio/udfs/tree/fb65aff/public/DuckDB_H3_Example/"
-    ).utils
+    con = duckdb.connect(config={"allow_unsigned_extensions": True})
```

Comment:
```suggestion
    con = duckdb.connect()
```

The "unsigned" config should no longer be needed when loading H3 from the community extensions.